extensions/bitwarden/src/components/searchVault/ItemActionPanel.tsx

```diff
@@ -16,10 +16,13 @@ import {
   CopyLoginUrisActions,
   CopyCustomFieldsActions,
   PasteTotpAction,
+  CopyPublicKeyAction,
```

Comment:
Just needs the exports on in the `index.ts`

```suggestion
  CopyPublicKeyAction,
  CopyKeyFingerprintAction,
  CopyPrivateKeyAction
```